scripts/reborn_webui_v2_live_qa/test_run_live_qa.py

```
@@ -252,6 +253,32 @@ def fake_capability_run_statuses(_reborn_home, capability_ids):
             str(result.details["error"]),
         )
 
+    def test_endpoint_status_routine_prompt_uses_real_endpoint(self):
+        captured: dict[str, object] = {}
+
+        async def fake_live_chat_case(_ctx, **kwargs):
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `fake_live_chat_case`

(ANN202)